src/releng_treestatus/releng_treestatus/__init__.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/python
```

Comment:
``#/usr/bin/python`` is not required since this are not scripts only ``# -*- coding: utf-8 -*-``